specs/packagemanager/PackageManagement.md

```diff
@@ -398,6 +400,24 @@ Is\*Provisioned\*() methods determine if the target is provisioned.
 
 These methods require administrative privileges.
 
+## 3.13. PackageValidator
+
+This API allows callers to verify that packages being processed by PackageDeploymentManager.AddPackageAsync
```

Comment:
Add*() has multiple overloads e.g. `AddPackageAsync()`, `AddPackageByUriAsync()`, `AddPackageSetAsync()` and ditto for Stage*().

RECOMMEND: Use more general phrasing e.g.

```
This API enables callers to verify packages processed by add and stage requests match the caller's expected package(s). This functionality is available via `PackageDeploymentManager` methods `Add*Async()`, `Ensure*Async()` and `Stage*Async()`.
```